psi4/driver/procrouting/__init__.py

```diff
@@ -31,3 +31,5 @@
 from .empirical_dispersion import EmpericalDispersion
 from . import dft_funcs
 from . import response
+
+from . import scf # FIXME! DONT DO THIS.
```

Comment:
btw, @dgasmith, what did you have in mind instead here?

Reply:
I think I was worried about fudging up the namespace. The name `scf` is a bit too generic I think. Perhaps something like `scf_procrouting` or some such.

Reply:
Got it. This, and your other points, should be cleaned up now.